spec/sign_up_spec.rb

```diff
@@ -0,0 +1,82 @@
+feature 'Signing up to Chitter' do
+
+  let!(:user) do
+    User.new(name: 'Katie', username: 'Smithy', email: 'K@email.com', password: 'secret', password_confirmation: 'secret')
```

Comment:
Metrics/LineLength: Line is too long. [122/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)